recipes/r-pubmedwordcloud/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set version = '0.3.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pubmedwordcloud
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/PubMedWordcloud_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/PubMedWordcloud/PubMedWordcloud_{{ version }}.tar.gz
+  sha256: 824f619b6e2bdc807a7bc4cba7ddaef06db8f342a3b7c8b3bec72a92af0bd7f1
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-rcolorbrewer
+    - r-rcurl
+    - r-xml
+    - r-stringr
+    - r-tm
+    - r-wordcloud
+  run:
+    - r-base
+    - r-rcolorbrewer
+    - r-rcurl
+    - r-xml
+    - r-stringr
+    - r-tm
+    - r-wordcloud
+
+test:
+  commands:
+    - $R -e "library('PubMedWordcloud')"           # [not win]
+    - "\"%R%\" -e \"library('PubMedWordcloud')\""  # [win]
+
+about:
+  home: http://felixfan.github.io/PubMedWordcloud/
+  license: GPL-2
```

```suggestion
  license: GPL-2.0-or-later
```

According to:
* https://cran.r-project.org/web/packages/PubMedWordcloud/index.html
* https://spdx.org/licenses/GPL-2.0-or-later.html